package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "asgardex",
   "productName": "ASGARDEX",
-  "version": "0.1.0",
+  "version": "0.0.1",
   "description": "WALLET AND EXCHANGE CLIENT FOR THORCHAIN",
-  "main": "public/electron.js",
+  "main": "index.js",
   "scripts": {
     "start:web": "craco start",
     "dev": "concurrently --kill-others --success first \"yarn:dev:*\"",
     "dev:app": "cross-env BROWSER=none yarn start:web",
-    "dev:electron": "wait-on http://localhost:3000 && nodemon -I --watch ./src/main/ --exec electron ./public/dev.js",
+    "dev:electron": "wait-on http://localhost:3000 && nodemon -I --watch ./src/main/ --exec electron .",
```

Comment:
`--exec electron .` -> `--exec electron ./index.js` (just to make it more clear for non JS users)

Reply:
unfortunately we can not do that for the [same issue](https://github.com/electron/electron/issues/7085#issuecomment-354305046) we discussed with you.
P.S. yep. i've tried to use `./index.js` directly and there is comparison with `electro`'s version
![image](https://user-images.githubusercontent.com/67793194/116727472-48e14c00-a9ed-11eb-82bc-ce02a085b167.png)